Error after update

```
TypeError: Cannot read property 'endOffset' of undefined
Occurred while linting E:\fg\fg\src\components\fg-new-collection1-
area.js:17
    at TemplateAnalyzer.getRawAttributeValue (E:\fg\fg\node_module
s\eslint-plugin-lit\lib\template-analyzer.js:98:21)
    at Object.enterElement (E:\fg\fg\node_modules\eslint-plugin-li
t\lib\rules\attribute-value-entities.js:43:59)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:159:29)
    at E:\fg\fg\node_modules\eslint-plugin-lit\lib\template-analyz
er.js:166:25
    at Array.forEach (<anonymous>)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:165:30)
    at E:\fg\fg\node_modules\eslint-plugin-lit\lib\template-analyz
er.js:166:25
    at Array.forEach (<anonymous>)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:165:30)
    at TemplateAnalyzer.traverse (E:\fg\fg\node_modules\eslint-plu
gin-lit\lib\template-analyzer.js:174:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! fg@1.0.0 lint:src: `eslint --config .eslintrc.json src/`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the fg@1.0.0 lint:src script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.
```

```
import { LitElement, html, customElement, property } from 'lit-element';
import { connect } from 'pwa-helpers/connect-mixin.js';
import { store } from '../store.js';

export @customElement('fg-new-collection1-area')
class FGNewCollection1Area extends connect(store)(LitElement)
{
```

That's the only lit thing's that whole file uses. So not sure what could be wrong.
The previous version of this plugin worked perfectly.

Unfortunatly it still fails with exactly same error with version 1.1.1.
Although slightly different place in the code.

```
TypeError: Cannot read property 'endOffset' of undefined
Occurred while linting E:\fg\fg\src\components\fg-new-collection1-
area.js:17
    at TemplateAnalyzer.getRawAttributeValue (E:\fg\fg\node_module
s\eslint-plugin-lit\lib\template-analyzer.js:101:21)
    at Object.enterElement (E:\fg\fg\node_modules\eslint-plugin-li
t\lib\rules\attribute-value-entities.js:43:59)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:162:29)
    at E:\fg\fg\node_modules\eslint-plugin-lit\lib\template-analyz
er.js:169:25
    at Array.forEach (<anonymous>)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:168:30)
    at E:\fg\fg\node_modules\eslint-plugin-lit\lib\template-analyz
er.js:169:25
    at Array.forEach (<anonymous>)
    at visit (E:\fg\fg\node_modules\eslint-plugin-lit\lib\template
-analyzer.js:168:30)
    at TemplateAnalyzer.traverse (E:\fg\fg\node_modules\eslint-plu
gin-lit\lib\template-analyzer.js:177:9)
npm ERR! code ELIFECYCLE
npm ERR! errno 2
npm ERR! fg@1.0.0 lint:src: `eslint --config .eslintrc.json src/`
npm ERR! Exit status 2
npm ERR!
npm ERR! Failed at the fg@1.0.0 lint:src script.
npm ERR! This is probably not a problem with npm. There is likely additional log
ging output above.
```

--- a/src/template-analyzer.ts
+++ b/src/template-analyzer.ts
@@ -151,11 +151,19 @@ export class TemplateAnalyzer {
     if (!element.sourceCodeLocation) {
       return null;
     }
+
+    const xAttribs = element['x-attribsPrefix'];
+    let originalAttr = attr.toLowerCase();
+
+    if (xAttribs && xAttribs[attr]) {
+      originalAttr = `${xAttribs[attr]}:${attr}`;
+    }
+
     if (element.attribs[attr] === '') {
       return '';
     }
 
-    const loc = element.sourceCodeLocation.attrs[attr.toLowerCase()];
+    const loc = element.sourceCodeLocation.attrs[originalAttr];
     let str = '';
 
     for (const quasi of this._node.quasi.quasis) {

--- a/src/test/rules/attribute-value-entities_test.ts
+++ b/src/test/rules/attribute-value-entities_test.ts
@@ -31,7 +31,8 @@ ruleTester.run('attribute-value-entities', rule, {
     {code: "html`<x-foo attr=${'>'}></x-foo>`"},
     {code: 'html`<x-foo attr="()"></x-foo>`'},
     {code: 'html`<x-foo attr></x-foo>`'},
-    {code: 'html`<svg viewBox="0 0 48 48"></svg>`'}
+    {code: 'html`<svg viewBox="0 0 48 48"></svg>`'},
+    {code: 'html`<svg xlink:href="abc"></svg>`'}
   ],
 
   invalid: [

